docs/diagrams/ModelClassDiagram.puml

```diff
@@ -14,8 +14,8 @@ Class UserPrefs
 
 Class UniqueEateryList
 Class Eatery
-Class Address
-Class Email
+Class Location
+Class Cuisine
 Class Name
 Class Phone
```

Comment:
This should be changed to `Price` instead.

Reply:
Fair enough for 1, we should finalize the scroll bar first